frontend/src/api/profile/getProfile.ts

```diff
@@ -0,0 +1,31 @@
+import { useQuery } from 'react-query';
+
+import { apiClient } from 'api/base';
+
+const useGetProfile = (accessToken: string | undefined) =>
+  useQuery('profile', async () => {
+    let profileResponse: {
+      position?: string;
+      firstName?: string;
+      lastName?: string;
+      skills?: Array<string>;
+      courses?: Array<string>;
+      avatar?: string;
+      birthday?: Date;
+      skype?: string;
+      error?: unknown;
+    };
+    try {
+      const response = await apiClient.get('/api/account/profile', {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
```

Comment:
Let's describe auth header inside `apiClient`. It's not a good idea to define it each time for each route. Just check is client authorized(at this moment always true) inside API client wrapper.